networking_nsxv3/api/rpc.py

```diff
@@ -33,32 +31,19 @@ def __init__(self, context):
         self.rpc = rpc.get_client(target)
 
     def _get_call_context(self, host=None):
-        # True - broadcast to all agents
-        # False - send only to the host
-        fanout = True if host is None else False
-
         topic = topics.get_topic_name(
-            topics.AGENT, nsxv3_constants.NSXV3, topics.UPDATE, host
-        )
+            topics.AGENT, nsxv3_constants.NSXV3, topics.UPDATE, host)
 
+        # fanout=True - broadcast to all agents, False only to the host
         return self.rpc.prepare(
             version=self.version,
             topic=topic,
-            fanout=fanout
-        )
+            fanout=True if host is None else False)
```

Comment:
Personally, I would prefer a boolean expression directly, e.g.
`fanout=(host is None)`

But that's a matter of taste. Not sure, if we have a convention for that.